examples/ensurance/evict-on-mem-usage-total/watermark.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: ensurance.crane.io/v1alpha1
+kind: NodeQOS
+metadata:
+  name: eviction-on-high-mem-usage
+spec:
+  nodeQualityProbe:
+    nodeLocalGet:
+      localCacheTTLSeconds: 60
+    timeoutSeconds: 10
+  rules:
+    - actionName: eviction
+      avoidanceThreshold: 2
+      metricRule:
+        name: memory_total_usage
+        value: 5000000000
```

Comment:
Can it add an explanatory note this is "5Gi". Users can not know the unit conversion size